src/index.html

```diff
@@ -2,12 +2,58 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+          name="viewport">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link href="https://fonts.googleapis.com"
+          rel="preconnect">
+    <link href="https://fonts.gstatic.com"
+          rel="preconnect">
+    <link href="https://fonts.googleapis.com"
+          rel="preconnect">
+    <link href="https://fonts.gstatic.com"
+          rel="preconnect">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link href="./styles/index.scss"
+          rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div class="card"
+         data-qa="card">
+      <img alt="Monitir picture"
+           class="card__img"
+           src="../src/images/imac.jpeg">
```

Comment:
<img width="1073" alt="Pasted Graphic" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/8a2d9d7b-b5fe-4131-89bb-23afaef74ef2">
fix it everywhere